src/storekit.cs

```diff
@@ -525,34 +545,51 @@ interface SKStoreProductViewController {
 		void LoadProduct (StoreProductParameters parameters, [NullAllowed] Action<bool,NSError> callback);
 	}
 
+	[Mac (11,0), NoTV, NoWatch]
```

Comment:
missing `[iOS]` attribute

Reply:
hmm... that was moved :(
I rather have more `#if XXX` than moving code (to preserve git history)

Reply:
Was added in iOS 6,0.

Reply:
yeah, larger diff (than required) due to code being moved